### PR TITLE
Post learning lunch

### PR DESCRIPTION
**Pull request by reginafcompton**
This PR customizes the style guide per our lunch discussion on Feb 21.

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,43 +1,36 @@
...
+* [Use semicolons most of the time](#use-semicolons-most-of-the-time)
 * [80 characters per line](#80-characters-per-line)
-* [Use single quotes](#use-single-quotes)
+* [Use single or double quotes](#use-single-or-double-quotes)
 * [Opening braces go on the same line](#opening-braces-go-on-the-same-line)
 * [Declare one variable per var statement](#declare-one-variable-per-var-statement)
 
 ### Naming Conventions
-* [Use lowerCamelCase for variables, properties and function names](#use-lowercamelcase-for-variables-properties-and-function-names)
-* [Use UpperCamelCase for class names](#use-uppercamelcase-for-class-names)
```
Comment (hancush) [inline Readme.md]: I think I'd leave this in there.

File: Readme.md [hunk 2]
```diff
@@ -288,22 +304,24 @@ if (a == '') {
 
 [comparisonoperators]: https://developer.mozilla.org/en/JavaScript/Reference/Operators/Comparison_Operators
 
-### Use multi-line ternary operator
+### Minimize use of the ternary operator
```
Comment (hancush) [inline Readme.md:326]: Thank you!

**Review by hancush — COMMENTED** [review]
Thank you so much for this! I think I'd leave the `UpperCamelCase` for classes (since it's also a Python convention, and doesn't require context switching). What do you think?

File: Readme.md (hunk 1 above)
Comment (reginafcompton) [reply Readme.md]: Sounds good to me

**Pull request merged**